package/common_init/overlay/etc/init.d/S81sbp_fileio_daemon_external

```diff
@@ -2,6 +2,7 @@
 
 FILEIO_DIR=/data/
 PERSIST_DIR=/persistent/
+FLAG_DIR=/etc/flags/
```

Comment:
We need this change to be in [board/piksiv3/rootfs-overlay/etc/init.d/S81sbp_fileio_daemon_firmware](https://github.com/swift-nav/piksi_buildroot/blob/master/board/piksiv3/rootfs-overlay/etc/init.d/S81sbp_fileio_daemon_firmware)